pkg/processor/service.go

```diff
@@ -120,7 +115,7 @@ func (s *Service) Create(
 	}
 
 	s.instances[instance.ID] = instance
-	measure.ProcessorsGauge.WithValues(name, t.String()).Inc()
+	measure.ProcessorsGauge.WithValues(name).Inc()
```

Comment:
Don't forget to change the `ProcessorsGauge` and remove the `type` label.

Reply:
Done.